pybossa/stats/gold.py

```diff
@@ -0,0 +1,128 @@
+"""
+This module should not depend on either the app or the request context
+"""
+from itertools import chain
+import logging
+
+import numpy as np
+
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+
+class NoAnswer(object):
```

Comment:
I don't understand the intent behind the `Answer` and `NoAnswer` classes. It looks like you want to wrap the task results in a standard interface, but I don't see what your code is adding to the built-in `list` and `dict` types.